pkg/commands/app.go

```diff
@@ -215,6 +216,12 @@ var (
 		EnvVars: []string{"TRIVY_K8S_NAMESPACE"},
 	}
 
+	reportFlag = cli.StringFlag{
+		Name:  "report",
+		Value: "json",
+		Usage: "specify a report format for the output. (json,summary default: json)",
```

Comment:
IMHO, `--report` is different from `--format`. `--report` is how we aggregate/summarize the result and `--format` is the format like `table` and `json`. So, `--report nsa --format table` and `--report nsa --format json` both should be available. Looks like they are mixing now.

Reply:
The default value of `--format` is `table` and `--report` is `summary`. There are two available combinations for now.

Default:

```
--report summary --format table
```

Also:
```
--report all --format json
```